Assets/MRTK/SDK/Experimental/Elastic/Scripts/ElasticsManager.cs

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Physics
+{
+    /// <summary>
+    /// ElasticsManager can be used to add elastics simulation to supporting components.
+    /// Call Initialize on manipulation start.
+    /// Call ApplyHostTransform to apply elastics calculation to target transform.
+    /// Elastics will continue simulating once manipulation ends through it's update function - 
+    /// to block the elastics auto update set EnableElasticsUpdate to false. 
+    /// </summary>
+    [HelpURL("https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/README_ObjectManipulator.html")] // todo help url
```

Comment:
Nit: It would probably be better to remove the wrong HelpURL - some TODOs have a way of remaining forever.

If you're going to do this in the next PR then it's fine, but if you think you could be 2+ away then it has a decent chance of getting forgotten.

Reply:
Ahhhh. I missed that one 😅. I'm gonna remove it for now. Gonna get docs going after we branch 😊